tox.ini

```diff
@@ -473,6 +473,9 @@ ignore_errors=True
 [mypy-packages.fetchai.*]
 ignore_errors=True
 
+[mypy-packages.valory.*]
+ignore_errors=True
```

Comment:
The repo had the same issue as in https://github.com/valory-xyz/governatooorr/issues/32.

Reply:
Don't the other repos have this issue? Aren't we using the same approach everywhere?

Reply:
I only found this on IEKit and governatooorr.